src/main/java/duke/Command.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
```

Comment:
You can add a JavaDoc to explain the purpose of this abstract class and how it can be extended by its child classes.